api/src/users_clock.js

```diff
@@ -82,7 +84,13 @@ async function usersWorker () {
 
 // Run
 if (require.main === module) {
-  usersWorker()
+  dbSettings.list().then(settings =>
+    // load the cache
+    settings.forEach(({ setting, value }) => {
+      cache.put(setting, value)
+    })
+  )
```

Comment:
This code is duplicated, so I would consider refactoring it to a single function. But actually the more I consider it, it would be better if it did not have exist at all. I think this implementation is pointing to an antipattern within the memory cache usage. If there is a cache miss, the memory cache could hit the models/settings module to attempt to restore the cache from the database (lazy loading). Pre-loading a cache manually,  and having to do so in multiple  places seems error prone and easy to forget again.

Reply:
Yes I agree, I think that manually loading the cache is an anti-pattern. I'll think of a better way here for a cache miss. The issue that i've been facing is mostly related to order of dependencies loaded in the database and so manually loading from the database was the easiest fix.

Reply:
@kamicut it appears memory-cache does not support lazy loading, however, it could be used wrapped in a closure like this:

https://gist.github.com/guidorice/adf0628871796be3324a54f3803725b9